package com.jg.workflow.exception;

/**
 * create by 2017/4/11.
 *
 * @author yimin
 */

public class TaskIsOver extends RuntimeException {
	public TaskIsOver(int taskId) {
		super("任务已经被执行完成.ID:" + taskId);
	}
}
